Userland/Libraries/LibTextCodec/Decoder.cpp

```diff
@@ -250,6 +253,28 @@ String UTF16BEDecoder::to_utf8(StringView input)
     return builder.to_string();
 }
 
+void UTF16LEDecoder::process(StringView input, Function<void(u32)> on_code_point)
+{
+    size_t utf16_length = input.length() - (input.length() % 2);
+    for (size_t i = 0; i < utf16_length; i += 2) {
+        u16 code_point = input[i] | (input[i + 1] << 8);
+        on_code_point(code_point);
+    }
+}
+
+String UTF16LEDecoder::to_utf8(StringView input)
+{
+    // Discard the BOM
+    auto bomless_input = input;
+    if (auto bytes = input.bytes(); bytes.size() >= 2 && bytes[0] == 0xFF && bytes[1] == 0xFE) {
+        bomless_input = input.substring_view(2);
+    }
```

Comment:
please omit curly braces

Reply:
Copied over from the UTF-16BE decoder, so I've updated that as well.